cli/schemas/config-file.v1.json

```diff
@@ -672,6 +672,10 @@
         }
       }
     },
+    "minimumDependencyAge": {
```

Comment:
I think we should consider adding `pm` key to house all of these installation settings